### PR TITLE
Updated run and run-parallel Makefile tasks

### PR DESCRIPTION
**Pull request by antoniocarlon**
As now we need to completely qualify the Task name, there is no need to hide the `tasks` part in the `module` Luigi parameter

### CODE REVIEW

**Comment by javitonino** [comment]
An idea, because I'm lazy. Now that the task is fully qualified, we can calculate the `--module` part automatically (it's the task name except the part after the last dot). That way we could do `make run tasks.es.ine.PopulationMetaWrapper` (and avoid writing `tasks.es.ine` twice). What do you think?

**Comment by antoniocarlon** [comment]
Avoid unnecesary typing? I'm in :+1:

File: Makefile [hunk 1]
```diff
@@ -107,17 +107,29 @@ kill:
...
 ifeq (run,$(firstword $(MAKECMDGOALS)))
-  # use the rest as arguments for "run"
-  RUN_ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  # https://stackoverflow.com/questions/10571658/gnu-make-convert-spaces-to-colons
+  space :=
+  space +=
+  # From word 2 to the end is the task
+  TASK := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  # Split the quelified task into words and remove the last word (task name) to get the module name
+  MOD_NAME := $(subst $(space),.,$(filter-out $(word $(words $(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))),$(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))), $(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))))
```
Comment (javitonino) [inline Makefile]: Wow! That Makefile syntax!

Trying to see it from the outside, I asked Google how we could run shell commands in here. You can do something like `MOD_NAME != command`

Maybe something like `echo $(TASK) | sed "s/^\(.*\)\..*$/\1/"`? Still not very readable, but at least is just a regex, not some custom `make` thing.

File: Makefile [hunk 2]
```diff
@@ -278,178 +286,118 @@ deps-tree:
 
 ### au
 au-all:
-	docker-compose run --rm bigmetadata luigi \
-	  --module tasks.au.data tasks.au.data.BCPAllGeographiesAllTables --year 2011 \
-	  --parallel-scheduling --workers=8
+	 au.data.BCPAllGeographiesAllTables --year 2011
```
Comment (javitonino) [inline Makefile]: Missing `make run-parallel`

**Comment by antoniocarlon** [comment]
Shame on me, damn copy-paste!

**Pull request merged**